server/base/src/test/java/org/apache/accumulo/server/security/handler/ZKAuthenticatorTest.java

```diff
@@ -144,14 +145,16 @@ public void testUserAuthentication() throws Exception {
     var instanceId = InstanceId.of("example");
     ZooSession zk = createMock(ZooSession.class);
     ServerContext context = MockServerContext.getWithMockZK(zk);
+    ZooCache zc = createMock(ZooCache.class);
     expect(context.zkUserPath()).andReturn(ZooUtil.getRoot(instanceId) + Constants.ZUSERS)
         .anyTimes();
     expect(zk.getChildren(anyObject(), anyObject())).andReturn(Arrays.asList(principal)).anyTimes();
     expect(zk.exists(matches(ZooUtil.getRoot(instanceId) + Constants.ZUSERS + "/" + principal),
         anyObject(Watcher.class))).andReturn(new Stat()).anyTimes();
-    expect(zk.getData(matches(ZooUtil.getRoot(instanceId) + Constants.ZUSERS + "/" + principal),
-        anyObject(), anyObject())).andReturn(newHash).once();
-    replay(context, zk);
+    expect(context.getZooCache()).andReturn(zc).anyTimes();
+    expect(zc.get(matches(ZooUtil.getRoot(instanceId) + Constants.ZUSERS + "/" + principal)))
```

Comment:
Is 'matches()' the same as 'eq()'? I don't think I've used that before. In any case, this doesn't seem to need a matcher since we're matching on an exact value.

```suggestion
    expect(zc.get(ZooUtil.getRoot(instanceId) + Constants.ZUSERS + "/" + principal))
```

Reply:
Removed these in a481f9b